addons/im_livechat/static/tests/qunit_suite_tests/components/chat_window_manager_tests.js

```diff
@@ -18,21 +19,18 @@ QUnit.module('chat_window_manager_tests.js', {
 QUnit.test('closing a chat window with no message from admin side unpins it', async function (assert) {
```

Comment:
This test has been adapted to use `pyEnv`, because change in `MockServer/_mockMailChannelChannelFetched` requires having a `mail.channel.partner` record being made automatically, and it's working with `pyEnv` but not with `this.data` .

I think it comes from `_mockCreate` not properly called with `this.data`, which itself doesn't auto-create `mail.channel.partner` from `MockServer/_mockMailChannel__inverseChannelPartnerIds`.